src/Main.elm

```diff
@@ -1,47 +1,141 @@
 module Main exposing (..)
 
 import Browser
-import Html exposing (Html, text, div, h1, img)
-import Html.Attributes exposing (src)
+import Html exposing (..)
+import Html.Attributes exposing ( href, class )
+import Html.Events exposing (..)
+import Http
+import RemoteData exposing (RemoteData, WebData)
+import Json.Decode as Decode exposing (Decoder, int, list, string)
+import Json.Decode.Pipeline exposing (required)
 
 
 ---- MODEL ----
 
+type alias News =
+    { by : String
+     , id : Int
+     , score : Int
+     , title : String
+    }
+
 
 type alias Model =
-    {}
+    { hackerNews : WebData (List News) }
 
+emptyModel : Model
+emptyModel =
+    { hackerNews = RemoteData.NotAsked }
 
 init : ( Model, Cmd Msg )
 init =
-    ( {}, Cmd.none )
+    ( emptyModel, Cmd.none )
 
 
 
 ---- UPDATE ----
 
 
 type Msg
-    = NoOp
-
+    = SendHttpRequest
+    | DataReceived (WebData (List News))
+
+dataDecoder: Decoder News
```

Comment:
The Json you get back has only IDs

![image](https://user-images.githubusercontent.com/125707/90173047-8002c500-ddac-11ea-93cf-b238a3e686f3.png)

So indeed it cannot find any "title" or "score".

So for now just grab that list. Next task would be to fetch the actual News by:

`"https://hacker-news.firebaseio.com/v0/item/" ++ itemId ++ ".json"`